spec/index.bs

```diff
@@ -6176,6 +6347,20 @@ dictionary GPURenderPassDescriptor : GPUObjectDescriptorBase {
     Issue(gpuweb/gpuweb#503): support for no attachments
 </div>
 
+<div algorithm="GPURenderPassDescriptor accessors" dfn-for=RenderPassDescriptor>
+    For a given {{GPURenderPassDescriptor}} value |descriptor|, the syntax:
+
+      - |descriptor|.`renderExtent` refers to
+        {{GPUTextureView/[[renderExtent]]}} of any {{GPUTextureView/[[descriptor]]}}
+        in either |descriptor|.{{GPURenderPassDescriptor/depthStencilAttachment}}/{{GPURenderPassDepthStencilAttachment/view}},
+        or any of the {{GPURenderPassColorAttachment/view}} in |descriptor|.{{GPURenderPassDescriptor/colorAttachments}}.
+
+    Issue: make it a define once we reference to this from other places
```

Comment:
You can put a placeholder ref at the bottom of the doc ("Temporary usages of non-exported dfns")